amoro-mixed-format/amoro-mixed-format-flink/amoro-mixed-format-flink-common/src/main/java/org/apache/amoro/flink/catalog/FlinkUnifiedCatalog.java

```diff
@@ -221,8 +226,25 @@ public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
     TableIdentifier tableIdentifier =
         TableIdentifier.of(
             unifiedCatalog.name(), tablePath.getDatabaseName(), tablePath.getObjectName());
+    if (!configuration.contains(TABLE_FORMAT)) {
```

Comment:
```suggestion
    if (!configuration.contains(TABLE_FORMAT) && ignoreIfExists && unfiiedCatalog.tableExists()) {
```

Reply:
Even when the user applies the "create table xxx()" statement to create a table, it is still necessary to validate the format of the table. So there is no need to add additional conditions.